docs/deprecated.md

```diff
@@ -61,7 +61,7 @@ The table below provides an overview of the current status of deprecated feature
 | Removed    | [logentries logging driver](#logentries-logging-driver)                                                                            | v24.0      | v25.0  |
 | Removed    | [OOM-score adjust for the daemon](#oom-score-adjust-for-the-daemon)                                                                | v24.0      | v25.0  |
 | Removed    | [Buildkit build information](#buildkit-build-information)                                                                          | v23.0      | v24.0  |
-| Deprecated | [Legacy builder for Linux images](#legacy-builder-for-linux-images)                                                                | v23.0      | -      |
+| Deprecated | [Legacy builder for Linux images](#legacy-builder-for-linux-images)                                                      [deprecated.md](deprecated.md)          | v23.0      | -      |
```

Comment:
?

Reply:
Hmm.. or may be I did a `CMD+V` in the wrong spot; could've been me for sure!